rpcs3/Emu/RSX/Overlays/overlay_perf_metrics.cpp

```diff
@@ -244,12 +245,10 @@ namespace rsx
 
 		void perf_metrics_overlay::set_detail_level(detail_level level)
 		{
-			if (m_detail == level)
```

Comment:
I don't think the changes in this function are in any way useful

Reply:
they aren't here but I included them anyway
they were when I had some update logic in there at the end (before I removed that code), only to realize that there was an early return
so it's to prevent that from happening again

Reply:
I don't see how this is relevant

Reply:
anyway I see it's called `framerate` in the rest of the code you've added so I'll revert

Reply:
not sure why it's such a big deal to you but sure.